### ISSUE TITLE
extremely unfortunate neptunus synergy

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1933]
the thing refuses to work with technology (at least in my case it was when I played lilith and had it)

disappointing af considering it's my favorite planetarium item

**Comment by ZMBanshee** [comment]
It just doesn't have a synergy, doesn't have much to do with Lilith.

**Comment by Lonrite** [comment]
Yeah that's another reason for me to avoid the item like a plague. Not only does it ruin dead eye | parasitoid I eye of belial now it's also this.

That item is only really good for devil enjoyers with their knife / brim loadouts i'd much rather always have tech 2 or tech 0.5, it's also nice with homing from angels ig but eh

**Comment by Lonrite** [comment]
Dead Eye technology is only not pathetic if you've already had the dmg boost stacked up to maximum beforehand, meaning it's impossible to ever wash it away by messing your accuracy up.

even then this is probably a bug but it should stay as a feature I like it a lot

**Comment by Rchardon** [comment]
> Yeah that's another reason for me to avoid the item like a plague. Not only does it ruin dead eye | parasitoid I eye of belial now it's also this.

Neptunus works perfectly with dead eye, parasitoid and eye of belial, are we talking about the same item ?
